test_framework/templates/3-nodes-k8s.yml

```diff
@@ -1,14 +1,18 @@
 kubernetes_version: v1.13.5-rancher1-2
 nodes:
   - address: K8S_CONTROLLER_IP
+    hostname_override: longhorn-tests-00 
```

Comment:
It's better for those to be replaced (as in a template) while running the script instead of hard coding. Otherwise, if we want more nodes or a different name defined in cloud-providers this will fail.